backend/bin/www

```diff
@@ -1,4 +1,3 @@
-
 #!/usr/bin/env node
```

Comment:
I got an error `npm ERR! code ELIFECYCLE` when running `npm run dev`. I could get it to work by removing this `#!/usr/bin/env node` line. I'm running on windows. Apart from that, I can see the "tech lady" example in my localhost:3001/users so I think everything works fine. If there's anything specific I should test, just let me know. I'm very unfamiliar with the backend :)

Reply:
Could you screenshot the whole error @dzldebby ? Thanks

Reply:
@thchia I'm not able to re-create the error by putting back that line, but I also did the steps listed in this stackoverflow, and the error I faced is similar to the OP's [https://stackoverflow.com/questions/42308879/how-to-solve-npm-error-npm-err-code-elifecycle]

Reply:
thanks for reviewing this PR @dzldebby.

did you run the migrations and seed? this PR gets rid of the `/users` route and instead replaces it with a `/students` route, so `localhost:3001/users` should no longer be a valid route.

first, you should create the databases `irrc_development` and `irrc_test`:

```
$ sudo -i -u postgres
postgres@server:~$ createdb irrc_development
postgres@server:~$ createdb irrc_test
postgres@server:~$ exit
```

then, run the migrations and seed:

```
$ npm run db:migrate
$ npm run db:seed
```

then try `npm run dev` and take a look at `localhost:3001/students`.

Reply:
@therealyingtong Got the following error at `npm run db:migrate`:
<img width="658" alt="Capture" src="https://user-images.githubusercontent.com/19401055/96460763-28725100-1256-11eb-8ec8-c6ec9a057cb3.PNG">

Reply:
Resolved by deleting the tables in irrc_development database and re-running `npm run dev`